Add attribute to help configure log channels

sunkan opened the pull request.

sunkan committed:
Add attribute to help configure log channels

bix0r commented on README.md:
````diff
@@ -136,3 +136,19 @@ $logger->alert('Db connect error', [
 ```
 
+
+## Using LogChannel with `stefna/di`
````
Perhaps explain what we mean be a "log channel"?

bix0r reviewed (commented):
Looks good. I wonder if it is easy to add a test for this? If not, then don't bother

sunkan committed:
Refactor ChannelWrapper to only set log channel if it's not part of context

This is to allow local override of log channel.

This can be needed if a class does say both payment and accounting
and you want messages log to separate channels for does

bix0r reviewed (approved):
Nice!
